constraint_utils/constraint_basics.py

```diff
@@ -191,12 +191,42 @@ def split_context_mt(raw_context_ht_path: str, coverage_ht_paths: Dict[str, str]
 
 def pre_process_data(ht: hl.Table, split_context_ht_path: str,
                      output_ht_path: str, overwrite: bool = False) -> None:
+    """ 
+    Join gnomAD and VEP context Tables.
+    
+    Function drops `a_index`, `was_split`, and`colocated_variants` annotations from gnomAD data.
+
+    :param ht: gnomAD exomes or genomes public Hail Table. 
+    :param split_context_ht_path: Path to VEP context Table.
+    :param output_ht_path: Path to output Table.
+    :param overwrite: Whether to overwrite existing data. Default is False.
+    """
     context_ht = hl.read_table(split_context_ht_path).drop('a_index', 'was_split')
     context_ht = context_ht.annotate(vep=context_ht.vep.drop('colocated_variants'))
     ht.annotate(**context_ht[ht.key], pass_filters=hl.len(ht.filters) == 0).write(output_ht_path, overwrite)
 
 
 def prepare_ht(ht, trimer: bool = False, annotate_coverage: bool = True):
+    """
+    Filters rows that have more than one base in 'ref' and 'alt' columns, removes rows with Ns, 
+    collapses strand, and annotates the input table. 
+
+    Function adds the following annotations:
+        - ref
+        - alt
+        - was_flipped
+        - methylation_level
+        - cpg
+        - transition
+        - variant_type
+        - variant_type_model
+        - exome_coverage
+   
+    :param ht: Input table to be annotated
```

Comment:
```suggestion
    :param ht: Input Table to be annotated.
```
(we typically capitalize Table, also adding period for consistency)